### DISCUSSION TITLE
关于 contributor 展示的问题

### DISCUSSION THREAD

**Discussion by wxsms** [vuejs-translations/docs-zh-cn#459] [General]
> 最新的文档/翻译贡献情况可以参阅 GitHub 提供的 [contributors](https://github.com/vuejs-translations/docs-zh-cn/graphs/contributors) 页面。

目前由于同步了英文仓库的历史记录，contributors 页面主要还是是英文版贡献者，与本仓库的实际贡献情况相去甚远。是否有必要想办法解决一下呢？

**Comment by veaba** [comment]
同 git 索引下，肯定是会合在一起展示的。

1、手动过滤，手动维护
2、通过三方，看能否做过滤：https://allcontributors.org/ 
3、自己写个服务？

**Comment by veaba** [comment]
我写一个工具吧，有点思路了。

**Comment by Jinjiang** [comment]
Done

https://github.com/vuejs-translations/docs-zh-cn/pull/479
